SpywareFilter/sections/tracking_servers.txt

```diff
@@ -3201,6 +3201,7 @@
 ! Admiral domains
 ! These domains are used for tracking purposes according to the data that is being sent there.
 ! More info: https://getadmiral.com/products/measure
+||movemeal.com^$third-party
```

Comment:
This rule already exists in this section.

Reply:
Removed.